scala-package/core/src/main/scala/org/apache/mxnet/FeedForward.scala

```diff
@@ -230,8 +230,11 @@ class FeedForward private(
       val padded = batch.pad
       val realSize = batchSize - padded
       for ((list, nd) <- outputs zip predExec.outputs) {
-        list += nd.slice(0, realSize).copy()
+        val ndSliced = nd.slice(0, realSize)
```

Comment:
I think we should add a comment explaining why id *has* to be done this way, and not the more compact way used before. Otherwise, someone may easily revert this change unknowingly in the future.